{"name": "risk user -- @1.4 cases", "status": "passed", "steps": [{"name": "Given risk makes get request: /reconciliation/all_users_data/xlsx", "status": "passed", "start": 1610717893790, "stop": 1610717894502}, {"name": "Then check response of request: You do not have permission to perform this action. is False", "status": "passed", "start": 1610717894502, "stop": 1610717894657}], "parameters": [{"name": "url", "value": "/reconciliation/all_users_data/xlsx"}, {"name": "method", "value": "get"}, {"name": "bool", "value": "False"}], "start": 1610717893788, "stop": 1610717894657, "uuid": "388c4ef6-33b2-488c-b40c-102a2a38544b", "historyId": "ad308c72d421c1e6a04fe8711c9cc284", "fullName": "features/II-Reconciliation/III-reconciliation_access.feature:risk user", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "reconciliation check access"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}